src/util/alloc/mod.rs

```diff
@@ -10,6 +10,7 @@ pub use allocator::Allocator;
 pub(crate) mod allocators;
 pub use allocators::AllocatorSelector;
 pub use allocators::Allocators;
```

Comment:
With this PR, the changes in https://github.com/mmtk/mmtk-core/pull/880 can be reverted.

Reply:
Done